articles/team-fit.md

```diff
@@ -0,0 +1,35 @@
+---
+authorName: Marko Simic
+authorGithubUsername: masizuehlke
+issue: 90
+title: Team Fit
+---
+# {{page.title}}
+
+> “The human race is filled with passion. And medicine, law, business, engineering, these are noble pursuits and necessary to sustain life. But poetry, beauty, romance, love, these are what we stay alive for.” John Keating (Robin Williams), Dead Poets Society
+
+Science tends to measure and quantify things. By making them concrete, tangible, it sets boundaries to itself. But passion breaks them. That’s why they work so well together and why they can’t “be” without each other. For engineer to perform beyond ordinary, an organization must provide a system that will foster the passion. That system is culture.
+
+Culture is group phenomenon. It manifests through shared values and behaviors and experienced through the norms and expectations of a group. When in harmony with personal traits, provides environment that empowers people to perform at their best. Degree of that harmony is called _team fit_.
+
+Software engineering is highly collaborative process. From its very beginning, in discovery phase, until its retirement we’re in constant contact with various stakeholders. Furthermore, as we take on more responsibility, collaboration become more important.
+
+Software is everywhere. Professional services takes us to diverse industry sectors. Creating solution that will meet end-user expectations, require from us to really understand the business, regardless of the role.
```

Comment:
I'd remove this paragraph.

Reply:
I'm fighting with this one.
You're the third reviewer who said that introduction could be shorter.
And, believe it or not this is almost already a half of original version :)
Reason why it's so extensive is that I wanted to give a context from which I analyzed competences. I'm afraid If I shorten it more, it'll become list without background.

Let me think....